unpackaged/config/trial/profiles/Admin.profile

```diff
@@ -21,61 +21,63 @@
+        <recordType>hed__Behavior_Involvement__c.Incident_Involvement</recordType>
     </layoutAssignments>
     <layoutAssignments>
-        <layout>Contact-%%%NAMESPACE%%%K12 Kit Student Layout</layout>
-        <recordType>Contact.%%%NAMESPACED_ORG%%%Student</recordType>
+        <layout>hed__Behavior_Involvement__c-K12 Kit Positive Behavior Involvement Layout</layout>
+        <recordType>hed__Behavior_Involvement__c.Positive_Behavior_Involvement</recordType>
     </layoutAssignments>
-     <layoutAssignments>
-        <layout>hed__Address__c-%%%NAMESPACE%%%K12 Kit Address Layout</layout>
+    <layoutAssignments>
+        <layout>hed__Behavior_Response__c-K12 Kit Behavior Response Layout</layout>
+        <recordType>hed__Behavior_Response__c.Incident_Response</recordType>
     </layoutAssignments>
     <layoutAssignments>
-        <layout>hed__Affiliation__c-%%%NAMESPACE%%%K12 Kit Affiliation Layout</layout>
+        <layout>hed__Behavior_Response__c-K12 Kit Positive Behavior Response Layout</layout>
+        <recordType>hed__Behavior_Response__c.Positive_Behavior_Response</recordType>
     </layoutAssignments>
     <layoutAssignments>
-        <layout>hed__Application__c-%%%NAMESPACE%%%K12 Kit Application Layout</layout>
+        <layout>Case-%%%NAMESPACE%%%K12 Kit Incident Layout</layout>
+        <recordType>Case.%%%NAMESPACED_ORG%%%Incident</recordType>
     </layoutAssignments>
     <layoutAssignments>
-        <layout>hed__Attendance_Event__c-%%%NAMESPACE%%%K12 Kit Attendance Event Layout</layout>
+        <layout>Case-K12 Kit Positive Behavior Layout</layout>
+        <recordType>Case.%%%NAMESPACED_ORG%%%Positive_Behavior</recordType>
     </layoutAssignments>
     <layoutAssignments>
-        <layout>hed__Attribute__c-%%%NAMESPACE%%%K12 Kit Credential Layout</layout>
-        <recordType>hed__Attribute__c.hed__Credential</recordType>
+        <layout>Contact-%%%NAMESPACE%%%K12 Kit Faculty Staff Layout</layout>
+        <recordType>Contact.%%%NAMESPACED_ORG%%%Faculty</recordType>
     </layoutAssignments>
     <layoutAssignments>
-        <layout>hed__Attribute__c-%%%NAMESPACE%%%K12 Kit Student Characteristic Layout</layout>
-        <recordType>hed__Attribute__c.hed__Student_Characteristic</recordType>
+        <layout>Contact-%%%NAMESPACE%%%K12 Kit Guardian Layout</layout>
+        <recordType>Contact.%%%NAMESPACED_ORG%%%Guardian</recordType>
     </layoutAssignments>
     <layoutAssignments>
-        <layout>hed__Behavior_Involvement__c-%%%NAMESPACE%%%K12 Kit Behavior Involvement Layout</layout>
+        <layout>Contact-%%%NAMESPACE%%%K12 Kit Student Layout</layout>
     </layoutAssignments>
     <layoutAssignments>
-        <layout>hed__Behavior_Response__c-%%%NAMESPACE%%%K12 Kit Behavior Response Layout</layout>
+        <layout>Contact-%%%NAMESPACE%%%K12 Kit Student Layout</layout>
+        <recordType>Contact.%%%NAMESPACED_ORG%%%Student</recordType>
```

Comment:
looks like we have 2 <layout>Contact-%%%NAMESPACE%%%K12 Kit Student Layout</layout>. One with record type and the other one without. Other one's on the line 76. Should there be just 1 @kdy618 ?

Reply:
@skosgi1 , good catch. Let me double check.